toy/toy.go

```diff
@@ -0,0 +1,44 @@
+package toy
+
+// Toy is exported type
+type Toy struct {
+	Name   string
+	Weight int
+	onHand int
+	sold   int
+}
+
+//New was exported
+func New(name string, weight int) *Toy {
+	return &Toy{
+		Name:   name,
+		Weight: weight,
+	}
+}
+
+//OnHand is exporetd
+func (t *Toy) OnHand() int {
+	return t.onHand
+}
+
+//copied from answers , not too clear, so I had to read
+
```

Comment:
Whats with the hanging comment and space. Put them properly, I dont know which comment or function this refers to???